src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -56,27 +57,40 @@ const useStyles = makeStyles(({ breakpoints, spacing }) => {
   };
 });
 
-export type ProjectListItemCardProps = ProjectListItemFragment & {
+export interface ProjectListItemCardProps {
+  project?: ProjectListItemFragment;
   className?: string;
-};
+  loading?: boolean;
```

Comment:
See now both are optional and we have to check for twice for each property. What we really want is one or the other. So you've either given the `project` or you haven't which means we are loading.
```tsx
const loading = !project;
```

Reply:
ok, got it. The reason why I did like that is because you mentioned about loading prop in ticket description.